CI/physmon/physmon_traccc_common.py

```
@@ -0,0 +1,172 @@
+
+        s.run()
+        del s
+
+        recStr = "_reconstruction_only" if actsDigi else ""
+        name = f"tracksummary_traccc_{platform + recStr}"
+        perf_file = tp / f"{name}.root"
+        assert perf_file.exists(), "Tracksummary file not found"
+        shutil.copy(perf_file, setup.outdir / f"{name}.root")
+
+        if not actsDigi:
+            name = f"performance_seeding_traccc_{platform}";
```

Ruff (0.8.0)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)